### ISSUE TITLE
请问，为什么执行python3.5 fcn.py -m test后Res目录下生成的Pred_**.png 都是黑色？

### ISSUE THREAD

**Issue by DarkFunct** [qhan1028/Fully-Convolutional-Networks#2]
执行python3.5 fcn.py -m test后Res目录下生成的inp_0.png pred_0.png 
为什么会一片黑呢？
![inp_0](https://user-images.githubusercontent.com/38775458/60186862-2fa26e00-985f-11e9-8451-773ad4d493a8.png)
![pred_0](https://user-images.githubusercontent.com/38775458/60186869-3204c800-985f-11e9-884d-e844286773e2.png)

**Comment by XTaiQue** [comment]
python3.5 fcn.py -m test
====================================================
> [Args] data_dir: Data_zoo/MIT_SceneParsing/
> [Args] model_dir: Model_zoo/
> [Args] logs_dir: logs/
> [Args] start_iter: 0
> [Args] test_dir: test
> [Args] debug: True
> [Args] batch_size: 2
> [Args] video: False
> [Args] res_dir: res/
> [Args] mode: test
> [Args] iter: 100000
> [Args] learning_rate: 0.0001
====================================================
WARNING: Logging before flag parsing goes to stderr.
W0626 21:32:03.037029 140115421640448 deprecation_wrapper.py:119] From fcn.py:164: The name tf.placeholder is deprecated. Ple

> [FCN] Setting up vgg initialized conv layers ...
W0626 21:32:03.364957 140115421640448 deprecation_wrapper.py:119] From fcn.py:89: The name tf.variable_scope is deprecated. P

W0626 21:32:03.365441 140115421640448 deprecation.py:506] From /usr/local/src/fully-Convolutional-Networks/tensorflow_utils.ped in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
W0626 21:32:03.365757 140115421640448 deprecation_wrapper.py:119] From /usr/local/src/fully-Convolutional-Networks/tensorflow

conv 1_1: (?, 224, 224, 64)
W0626 21:32:03.374490 140115421640448 deprecation_wrapper.py:119] From /usr/local/src/fully-Convolutional-Networks/tensorflow

W0626 21:32:03.375883 140115421640448 deprecation_wrapper.py:119] From /usr/local/src/fully-Convolutional-Networks/tensorflow

conv 1_2: (?, 224, 224, 64)
W0626 21:32:03.418985 140115421640448 deprecation_wrapper.py:119] From /usr/local/src/fully-Convolutional-Networks/tensorflow

pool 1  : (?, 112, 112, 64)
conv 2_1: (?, 112, 112, 128)
conv 2_2: (?, 112, 112, 128)
pool 2  : (?, 56, 56, 128)
conv 3_1: (?, 56, 56, 256)
conv 3_2: (?, 56, 56, 256)
conv 3_3: (?, 56, 56, 256)
conv 3_4: (?, 56, 56, 256)
pool 3  : (?, 28, 28, 256)
conv 4_1: (?, 28, 28, 512)
conv 4_2: (?, 28, 28, 512)
conv 4_3: (?, 28, 28, 512)
conv 4_4: (?, 28, 28, 512)
pool 4  : (?, 14, 14, 512)
conv 5_1: (?, 14, 14, 512)
conv 5_2: (?, 14, 14, 512)
conv 5_3: (?, 14, 14, 512)
conv 5_4: (?, 14, 14, 512)
----------------------------------------------------
conv 5_3: (?, 14, 14, 512)
W0626 21:32:04.171463 140115421640448 deprecation_wrapper.py:119] From /usr/local/src/fully-Convolutional-Networks/tensorflow

pool 5  : (?, 7, 7, 512)
W0626 21:32:04.172459 140115421640448 deprecation_wrapper.py:119] From /usr/local/src/fully-Convolutional-Networks/tensorflow

conv 6  : (?, 7, 7, 4096)
W0626 21:32:04.199513 140115421640448 deprecation.py:506] From fcn.py:105: calling dropout (from tensorflow.python.ops.nn_ops
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
conv 7  : (?, 7, 7, 4096)
conv 8  : (?, 7, 7, 2)
conv t1 : (?, 14, 14, 512)
fuse 1  : (?, 14, 14, 512)
conv t2 : (?, 28, 28, 256)
fuse 2  : (?, 28, 28, 256)
conv t3 : (?, ?, ?, 2)
W0626 21:32:04.292671 140115421640448 deprecation.py:506] From fcn.py:146: calling argmax (from tensorflow.python.ops.math_op
Instructions for updating:
Use the `axis` argument instead
prediction: (?, ?, ?)
W0626 21:32:04.293940 140115421640448 deprecation.py:506] From /usr/local/lib/python3.5/dist-packages/tensorflow/python/util/emoved in a future version.
Instructions for updating:
Use the `axis` argument instead
====================================================
W0626 21:32:04.312700 140115421640448 deprecation_wrapper.py:119] From fcn.py:211: The name tf.GPUOptions is deprecated. Plea

> [FCN] Setting up Saver...
W0626 21:32:04.321624 140115421640448 deprecation_wrapper.py:119] From fcn.py:216: The name tf.train.Saver is deprecated. Ple

W0626 21:32:04.412137 140115421640448 deprecation_wrapper.py:119] From fcn.py:217: The name tf.summary.FileWriter is deprecat

> [FCN] Initialize variables... 1308.1064 ms
==================================================== [test]
> image: 352624197101054x.jpggpg
> [FCN] Test: 0, Name: 350823199311202, 559.7322 ms
> [FCN] Test: 1, Name: 3526241948020421X, 487.9470 ms
> [FCN] Test: 2, Name: 3526241963157119_, 486.5894 ms
> [FCN] Test: 3, Name: 35010419720529005X, 486.5989 ms
> [FCN] Test: 4, Name: 3526241976165489, 485.2354 ms
> [FCN] Test: 5, Name: 3526241963157119(2), 486.8818 ms
> [FCN] Test: 6, Name: 3526249461208342X, 486.7593 ms
> [FCN] Test: 7, Name: 352624310714632, 485.2964 ms
> [FCN] Test: 8, Name: 352624197101054x, 486.2288 ms

**Comment by qhan1028** [comment]
出來的結果是0或1
所以看起來會是一片黑
你可用numpy array看看prediction
